app/macro.js

```diff
@@ -0,0 +1,6 @@
+import { writable } from 'ember-macro-helpers';
+import { translationMacro } from 'ember-intl';
+
+export const t = function() {
+  return writable(translationMacro(...arguments));
```

Comment:
Because the computed property that was created with `i18n`'s `t` there was existing logic that allowed calling `.set()` and replacing the existing computed property. However, in `ember-intl` this is set to a read-only property. This wrapper ensures that the `translationMacro` is wrapped and is `writable`.